### DISCUSSION TITLE
Loading scripts at startup.

### DISCUSSION THREAD

**Discussion by Hairo** [CEXT-Dan/PyRx#70] [Q&A] [answered]
Any way to load python scripts at startup like autolisp scripts?

**Comment by gswifort** [comment, marked as answer]
`%localappdata%\Programs\PyRx\Bin\pyrx_onload.py` -> `OnPyInitApp()` -> `Ap.Application.loadPythonModule`

**Comment by Hairo** [comment]
Thanks, can assume this file will be overwritten in a new update/reinstall, right?

Just something to keep in mind when updating.

**Reply by CEXT-Dan** [reply]
Hopefully this won’t be the case for ever. If I can figure out how to tell the installer to ignore existing files.  
I will remove the note on the releases once it's fixed

**Reply by CEXT-Dan** [reply]
Oh, I forgot to mention, there’s timing issues using the lisp functions, in onload, for the clones. AutoCAD works fine. Post an issue if you run into something wonky
